openedx/features/content_tagging/api.py

```diff
@@ -80,6 +81,77 @@ def get_taxonomies_for_org(
             yield taxonomy.cast()
 
 
+# Experiment
+def get_taxonomies_for_org_queryset(
```

Comment:
FYI @rpenido -- you don't have to write this part anymore, we've done it here: https://github.com/openedx/edx-platform/pull/32661/files#diff-0ed9e2727b75df5366745749e3b39718ffbefe4b60289d80feab3b0019feb66bR81-R100

Reply:
Also fyi, https://github.com/openedx/edx-platform/pull/32661 is merged (yay!), so you can rebase this on latest openedx:master and open a new PR against the upstream repo.

Reply:
Yay! \o/
Tomorrow I will rebase and create a new PR against upstream.